tom_nonlocalizedevents/views.py

```diff
@@ -16,6 +17,11 @@
 from tom_nonlocalizedevents.models import EventCandidate, EventLocalization, NonLocalizedEvent
 from tom_nonlocalizedevents.serializers import (EventCandidateSerializer, EventLocalizationSerializer,
                                                 NonLocalizedEventSerializer)
+from tom_nonlocalizedevents.alertstream_handlers import gw_event_handler
+
+
+logger = logging.getLogger(__name__)
+#logger.setLevel(logging.DEBUG)
```

Comment:
I think lint would like a space after the `#` here.

Reply:
That's a polite way of saying remove the commented out debugging line